.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This seems kind of random
I don't think adding any IDE-configs to code-base is a good idea.
Conflicts with individual setups are just waiting to happen.

_From what I've learned it considered a best practice to have a global git ignore in the git-setup to ignore all IDE config-dirs such as .vscode & .idea_

Reply:
alright I keep my amazing search filter to myself and add it to `gitignore` some other time.

Reply:
I don't think the world is ready for it just yet 😹